database/migrations/2020_12_09_000004_add_missing_indexes.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class AddMissingIndexes extends Migration
+{
+    public function up(): void
+    {
+        if (DB::connection()->getDriverName() !== 'mysql') {
+            throw new \InvalidArgumentException("MySQL is the only supported driver for this package.");
+        }
```

Comment:
I'm starting to think that every migration is not the place to check this. If the first migration threw this exception, you either fixed something or abandoned this as an option. If you changed databases between the first migration and this migration, you must have figured out how to make not MySQL work?

Reply:
If nothing else, this is repeated several times and could be DRY'd up.

Reply:
I think just removing it entirely and just saying we only support mysql (in readme) works for me.

Reply:
Same end result as far as I can see, an exception gets thrown that tells you what needs to be done. You decide if how you want to handle it. It's just either a PDO exception from an unsupported database operation, or an invalid argument.

Reply:
Okay - I cleared them out and updated readme.